### PR TITLE
ELEMENTS-902: Add a default time option to nuxeo-date-picker (Backport 9.10)

### PR DESCRIPTION
**Pull request by semisse**

### CODE REVIEW

File: test/nuxeo-date-picker.html [hunk 1]
```diff
@@ -112,6 +112,15 @@
       expect(getInput(element).value).to.be.equal('');
     });
 
+    test('the input changes takes default time into account', () => {
```
Comment (guirenard) [inline test/nuxeo-date-picker.html]: We don't use es6 arrow function for 9.10. Lint check will not pass

File: widgets/nuxeo-date-picker.html [hunk 2]
```diff
@@ -211,6 +216,16 @@
         if (this._inputValue !== null && !this._preventInputUpdate) {
           var date = moment(this._inputValue);
           if (date.isValid()) {
+            if (this.defaultTime) {
+              const time = moment(this.defaultTime, 'HH:mm:ss');
```
Comment (guirenard) [inline widgets/nuxeo-date-picker.html]: same same no es6

**Comment by guirenard** [comment]
:+1:

**Comment by Gabez0r** [comment]
👍

**Pull request merged**